tilings/strategies/pointing.py

```diff
@@ -0,0 +1,123 @@
+"""
+The directionless point placement strategy that is counted
+by the 'pointing' constructor.
+"""
+from typing import Dict, Iterator, Optional, Tuple
+
+from comb_spec_searcher import Strategy
+from comb_spec_searcher.exception import StrategyDoesNotApply
+from permuta.misc import DIR_NONE
+from tilings import GriddedPerm, Tiling
+from tilings.algorithms import RequirementPlacement
+
+from .unfusion import DivideByN, ReverseDivideByN
+
+
+class PointingStrategy(Strategy[Tiling, GriddedPerm]):
+    def can_be_equivalent(self) -> bool:
+        return False
+
+    def is_two_way(self, comb_class: Tiling) -> bool:
+        return True
+
+    def is_reversible(self, comb_class: Tiling) -> bool:
+        return True
+
+    def shifts(
+        self,
+        comb_class: Tiling,
+        children: Optional[Tuple[Tiling, ...]],
+    ) -> Tuple[int, ...]:
+        if children is None:
+            children = self.decomposition_function(comb_class)
+        assert children is not None
+        return tuple(0 for _ in children)
+
+    def decomposition_function(self, comb_class: Tiling) -> Tuple[Tiling, ...]:
+        cells = comb_class.active_cells - comb_class.point_cells
```

Comment:
This is slightly wrong, should only subtract point cells which are in their own row and column. Christian will fix this before merging